src/ts/platforms/google-play/googleplay-bridge.ts

```diff
@@ -319,7 +319,7 @@ namespace CdvPurchase {
             function extendAdditionalData(ad?: CdvPurchase.AdditionalData): AdditionalData {
                 const additionalData: AdditionalData = ensureObject(ad?.googlePlay);
                 if (!additionalData.accountId && ad?.applicationUsername) {
-                    additionalData.accountId = Utils.md5(ad.applicationUsername);
```

Comment:
Why are you removing the encryption?

Reply:
We need to be able to access the data when restoring a purchase. This information is then sent to the server to identify the purchase again and credit it to the user.